docs/content_management/images/images.md

```diff
@@ -231,8 +231,8 @@ a custom extension that lets you display and download files in your templates.
 First, enable adding SVG files to content by removing them from the blacklist 
 of allowed MIME types.
 
-To do it, comment out the relevant line under `ibexa.site_access.config.default.io.file_storage.file_type_blacklist`
-in `Core/Resources/config/default_settings.yml`.
+To do it, overwrite `ibexa.site_access.config.default.io.file_storage.file_type_blacklist` defined in `Core/Resources/config/default_settings.yml` so that `svg` is removed from the blacklist.
+You can do it per SiteAccess or SiteAccess group by using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).
```

Comment:
```suggestion
You can do it per SiteAccess or SiteAccess group by using [SiteAccess-aware configuration](siteaccess_aware_configuration.md).
```
@reithor we are using a plugin that allows linking by file name without specifying complete relative path. The link current may be working locally for you if you do not have that plugin installed, but that is why the check is failing in the PR.